<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindImagesWindow</class>
 <widget class="QWidget" name="FindImagesWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Images</string>
  </property>
  <widget class="QPushButton" name="FindImagesBtn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Images</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DirectoryPathTextBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListView" name="ClassesToFindListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>391</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fruit classes to find:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Directory path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseDirectoryBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Found images:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image preview:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImagePreviewLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>No Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="FoundImagesNumberLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QListView" name="FoundImagesListView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>391</width>
     <height>391</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="SearchingForImagesLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Searching for images...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
